server/app/api/__init__.py

```diff
@@ -22,7 +22,7 @@ def wrapper(*args, **kwargs):
             if response.status_code >= 400:
                 return response
             for k, v in cache_kwargs.items():
-                setattr(cache_control, k, v)
+                setattr(response.cache_control, k, v)
```

Comment:
Reverting to code as it was [previously](https://github.com/chanzuckerberg/single-cell-explorer/blob/8456f7eb5715077ece01ca134017a80a97f5f56f/server/app/app.py)